db/range_del_aggregator.cc

```diff
@@ -412,7 +412,6 @@ void RangeDelAggregator::AddToBuilder(
 
   // Note the order in which tombstones are stored is insignificant since we
   // insert them into a std::map on the read path.
-  bool first_added = false;
```

Comment:
could you have just moved this one line lower? Within each stripe the tombstones should be sorted. So we only need to consider the first tombstone in each stripe.